app/workers/publishing_api_notifier.rb

```diff
@@ -5,8 +5,8 @@ def perform(edition_id, update_type = "normal")
     edition = Edition.find(edition_id)
```

Comment:
Also the update_type should not default to "normal", as that is not a valid value. Probably should be "minor".